File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/pattern6.java
import java.util.Scanner;

public class pattern6 {
    public static void main(String[] args) {
        try (Scanner read = new Scanner(System.in)) {
            int n = read.nextInt();
            printTriangle(n);
        }
    }

    public static void printTriangle(int n) {
        // code here
        for (int i = 1; i <= n; i++) {
            for (int j = n; j > i; j--) {
                System.out.print(" ");
            }
            for (int k = 1; k <= i; k++) {
                System.out.print("*");
            }
            for (int a = 1; a < i; a++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}

File: SecondYear/swarnkarvikash_VikashKumarSwarnkar_2226csit1034_2/Week_1/Pattern6.cpp
class Solution {
  public:
    void printTriangle(int n) {
        // code here
        for(int i=1;i<=n;i++){
            for(int j=1;j<=(n-i);j++){
                cout<<" ";
            }
           
        for(int k=1;k<=(i-1);k++){
            cout<<"*";
        }
        for(int l=1;l<=i;l++){
            cout<<"*";
        }
        cout<<endl;
        }
    }
};